glitch/analysis/security.py

```diff
@@ -72,34 +129,18 @@ def __check_keyvalue(self, c: CodeElement, name: str,
             errors += self.check_element(value, file)
             value = repr(value)
 
-        try:
-            if (re.match(SecurityVisitor.__URL_REGEX, value) and
-                ('http' in value or 'www' in value) and 'https' not in value):
-                errors.append(Error('sec_https', c, file, repr(c)))
+        if self.__is_http_url(value):
+            errors.append(Error('sec_https', c, file, repr(c)))
 
-            parsed_url = urlparse(value)
-            if parsed_url.scheme == 'http' and \
-                    parsed_url.hostname not in SecurityVisitor.__URL_WHITELIST:
-                errors.append(Error('sec_https', c, file, repr(c)))
-        except:
-            # The url is not valid
-            pass
-
-        if re.match(r'^0.0.0.0', value):
+        if re.match(r'(?:https?://|^)0.0.0.0', value) or\
+                (name == "ip" and (value =="*" or value=='::')) or\
+                (name == "bind_ip_all" and value):
```

Comment:
I think the "ip" keyword is pretty generic, however the "bind_ip_all" seems more specific. If the keywords are specific to Docker, they should be moved to the config (or create an additional SmellChecker but I think the config is a better approach here).

Reply:
The bind_ip_all is a parameter for some commands, It may exists in equivalent commands in the other technologies, but I'm not sure

Reply:
Ye, I thought so. It is better to change it to a config for now.

Reply:
I've moved it to a config